Tests/E2E/OTAAJoinTest.cs

```diff
@@ -234,7 +234,7 @@ private async Task Test_OTAA_Join_Send_And_Rejoin_With_Custom_RX2_DR(TestDeviceI
             // Checking than the communication occurs on DR 4 and RX2 as part of preferred windows RX2 and custom RX2 DR
             await AssertUtils.ContainsWithRetriesAsync(x => x.StartsWith("+CMSG: RXWIN2", StringComparison.Ordinal), ArduinoDevice.SerialLogs);
             // this test has a custom datarate for RX 2 of 3
-            const string logMessage2 = "\"Rx2\":{\"Item1\":3";
+            const string logMessage2 = "\"Rx2\":{\"DataRateIndex\":3";
```

Comment:
Isn't is serialized as `DataRate` instead of `DataRateIndex` (if it comes from the record)?

Reply:
Indeed. Was just making sure the reviewer was paying attention. 🤣

Reply:
Fixed with ba825bdb02c86bc711f8bef1d1baad40c42d5d4e.